conf.py

```diff
@@ -113,7 +114,7 @@
 # built documents.
 #
 # The short X.Y version.
-version = "2021"
+version = str(date.today().year)
```

Comment:
I think during beta we probably actually want the version to reflect the next season year.

Maybe grab the year from the most recent git tag?

Reply:
is [this](https://github.com/robotpy/robotpy-docs/pull/50/commits/22832c75512150e97750bf0b8505fe2c2b3cfb2f#diff-6e03d3d89630aba3403afea3514a3ac5948e70d1ced091012e56446f614da340R118) what you meant?